seed/static/seed/js/controllers/sensor_readings_upload_modal_controller.js

```diff
@@ -113,9 +110,16 @@ angular.module('BE.seed.controller.sensor_readings_upload_modal', [])
 
       var show_confirmation_info = function () {
         uploader_service.sensor_readings_preview($scope.file_id, $scope.organization_id, $scope.view_id).then(function (result) {
+          var addtional_columnDefs = [
+            {
+              field: 'exists',
+              enableHiding: false
+            }
+          ]
+
           $scope.proposed_imports_options = {
             data: result,
-            columnDefs: base_sensor_readings_col_defs,
+            columnDefs: [...base_sensor_readings_col_defs, ...addtional_columnDefs],
```

Comment:
are the `...` correct?

Reply:
It's [spread](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#spread_in_array_literals) syntax.  🪄